### PR TITLE
Add rebuild and clean up instructions

### PR DESCRIPTION
**Pull request by krzysztofwolski**
Here are some one liners I use for fast environment tear down.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,58 @@ You can find the latest version of Saleor, storefront and dashboard in their ind
...
+<p>
+
+```
+$ docker-compose stop && docker-compose rm -f && docker-compose build && docker-compose run --rm api python3 manage.py migrate && docker-compose run --rm api python3 manage.py collectstatic --noinput && docker-compose run --rm api python3 manage.py populatedb --createsuperuser
+```
+
+</p>
+</details>
+
+### Still no available space
```
Comment (korycins) [inline README.md:108]: IMHO I would move it between 5th and `One-line solution`. `One-line solution` is section with some nice trick for docker but not directly connected with the disc space issue (I mean if points 1-4 didn't work for me, this command won't help me).

File: README.md [hunk 2]
```diff
@@ -68,6 +68,58 @@ You can find the latest version of Saleor, storefront and dashboard in their ind
...
+```
+
+4. Build fresh containers 
+```
+docker-compose build
+```
+
+5. Now you can run fresh environment using commands from `How to run it?` section. Done!
+
+### One-line solution
```
Comment (korycins) [inline README.md:95]: Should it be a part of this section? For me, it is more like helpful trick for managing docker containers

File: README.md [hunk 3]
```diff
@@ -68,6 +68,58 @@ You can find the latest version of Saleor, storefront and dashboard in their ind
...
+```
+
+</p>
+</details>
+
+### Still no available space
+
+If you are getting issues with lack of available space, consider prunning your docker cache:
+
+**Warning!** To get more info whats will be deleted, go to: https://docs.docker.com/engine/reference/commandline/system_prune/
```
Comment (tomaszszymanski129) [inline README.md:112]: Maybe ... more info on what will get deleted ...

File: README.md (hunk 2 above)
Comment (patrys) [reply README.md:95]: I'm not sure we should be posting this in our README, it's a hack that will lead to someone losing their database that for some reason had production data.

File: README.md (hunk 1 above)
Comment (krzysztofwolski) [reply README.md:108]: It was moved on purpose - pruning will remove your local cache. If not needed, it should not be run.

File: README.md [hunk 4]
```diff
@@ -68,6 +68,51 @@ You can find the latest version of Saleor, storefront and dashboard in their ind
...
+```
+
+2. Remove existing volumes
+
+**Warning!** Proceeding will remove also your database container! If you need existing data, please remove only services which cause problems! https://docs.docker.com/compose/reference/rm/
+```
+docker-compose rm
+```
+
+4. Build fresh containers
```
Comment (patrys) [inline README.md:88]: Where's number 3?

Comment (krzysztofwolski) [reply README.md:88]: ??????
4. PROFIT

Comment (krzysztofwolski) [reply README.md]: @patrys fixed :)

**Pull request merged**